src/components/common/ViewerSidebar.jsx

```diff
@@ -1239,6 +1238,7 @@ class ViewerSidebar extends Component {
                             {markerTable}
                             {legendTable}
                             {downloadSubLoomButton()}
+                            <GProfilerModal featureMetadata={md} />
```

Comment:
This still shows on the regulon tab but won't work because there is no sortable column. Two solutions:
1) Disable on regulon tab
2) Allow sorting by gene symbol if not other columns are available

Reply:
I think 1. makes more sense

Reply:
Actually I will enable it only for Clustering result which is more generic